IntelliTect.TestTools.Selenate.Tests/IntelliTect.TestTools.Selenate.Tests.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
Should this enable nullable reference types or warnings as errors?

Reply:
Completed when you called this out on a different project in the solution.